tools/azsdk-cli/Azure.Sdk.Tools.Cli/Models/Responses/TypeSpec/TypeSpecAuthoringResponse.cs

```diff
@@ -0,0 +1,72 @@
+using System.Text;
+using System.Text.Json.Serialization;
+using Azure.Sdk.Tools.Cli.Models.AiCompletion;
+using Azure.Sdk.Tools.Cli.Tools.TypeSpec;
+
+namespace Azure.Sdk.Tools.Cli.Models.Responses.TypeSpec
+{
+    public class TypeSpecAuthoringResponse : CommandResponse
+    {
+        [JsonPropertyName("is_successful")]
+        public bool IsSuccessful { get; set; }
```

Comment:
I checked other tools, they are using `result` property. Do you think we could use it?
<img width="860" height="927" alt="image" src="https://github.com/user-attachments/assets/6b64dac0-cbd2-40a8-a637-6a2d0e00dc86" />

Reply:
ok, in the spec, I changed to `result` as tool's output. Please let me know if you have any concern. Thanks. https://github.com/Azure/azure-sdk-tools/pull/12717/files#diff-16377f930533350031133c6cb8010cbef2eccd2edb0a13006d397eb2930bb4b1R319

Reply:
hello @haolingdong-msft I prefer not to use `result`, and in output json, it still use 'is_successful' as `TspToolResponse`. `Result` may not limited to success or failure,